hack/generate-manifest.sh

```diff
@@ -28,7 +28,8 @@ Generate a YAML manifest for Antrea using Kustomize and print it to stdout.
         --cloud                       Generate a manifest appropriate for running Antrea in Public Cloud
         --ipsec                       Generate a manifest with IPSec encryption of tunnel traffic enabled
         --proxy                       Generate a manifest with Antrea proxy enabled
-        --np                          Generate a manifest with Namespaced Antrea NetworkPolicy CRDs and ClusterNetworkPolicy related CRDs enabled
+        --np                          Generate a manifest with ClusterNetworkPolicy and AntreaNetworkPolicy feature gate enabled
+        --core-crd                    Generate a manifest with ExternalEntity CRD enabled
```

Comment:
I didn't review the whole PR, but could you clarify why we need a new switch here, and we can't just use `--np` to enable the "core" CRDs? Isn't CNP / ANP the only "consumer" of these CRDs right now?

Reply:
This is because per @abhiraut's suggestion we are first trying to merge AntreaNetworkPolicy without ExternalEntity support, so I'm trying to not include ExternalEntity CRD in yamls by default except for explicit testing

Reply:
i think we will support EE by end of this release.. so we can keep a single flag.. i was thinking to ensure validation schema does not allow EE in the yaml unless we have support for it

Reply:
Done.